docs/ref/checks.txt

```diff
@@ -84,6 +84,14 @@ Django's system checks are organized using the following tags:
 * ``templates``: Checks template related configuration.
 * ``caches``: Checks cache related configuration.
 * ``urls``: Checks URL configuration.
+* ``database``: Checks any database-related connection issues. Database checks
```

Comment:
This is not quite in agreement with the code. Connection issues are not checked AFAICT, and in fact nothing is checked except on MySql. Maybe "Checks database-related configuration issues"?